lib/get-html-headers.js

```diff
@@ -35,8 +35,18 @@ function rankify(headers, max) {
     })
 }
 
+// Remove code blocks so that false headers are not parsed
+function stripMDCodeBlocks(content) {
+  return content
+    .replace(/```([\s\S]*?)```/g, '') // must be before single backtick check
+    .replace(/`([\s\S]*?)`/g, '')
+    .replace(/<code>([\s\S]*?)<\/code>/g, '')
+    .replace(/<pre>([\s\S]*?)<\/pre>/g, '');
```

Comment:
At a guess it was probably the author's habit given . doesn't catch \n but in this case it already has newline removed so it could be changed back to just . if there is an assumption \n will never ever be passed in.